src/neuroconv/datainterfaces/ecephys/blackrock/blackrockdatainterface.py

```diff
@@ -22,8 +18,6 @@
 class BlackrockRecordingExtractorInterface(BaseRecordingExtractorInterface):
     """Primary data interface class for converting a BlackrockRecordingExtractor."""
 
-    RX = BlackrockRecordingExtractor
```

Comment:
oh wait, I don't know about this. Are you making this an instance attribute instead of a class attribute?

Reply:
It's always defined as a class attribute, but with lazy imports the default of `None` can only ever be set to the corresponding appropriate class after the external dependency has loaded, which is now occurring only when the object is initialized (that's the 'lazy' part - otherwise, setting it at the usual level of the class definition occurs simultaneous to the import of that class).

So we can either set it alongside `__init__` or at the `__new__` for the class - both have the same effect of triggering the import of the external dependency only when initializing the interface, and not otherwise.

Reply:
https://medium.com/python-features/class-vs-instance-variables-8d452e9abcbd

Reply:
hmm, I like the idea of being able to call `BlackrockRecordingExtractorInterface.RX` on the *class* not an instance of the class

Reply:
Another alternative could be setting the *name* (as as string) of the extractors class at that DataInterface class attribute level, which would have the same effect as we've always had for producing a corresponding associated name. But setting the *actual* extractor class as an *actual* class attribute at the same time you define the class itself is a non-lazy operation that occurs whenever you import the class.

Reply:
I also tried finding some ways that could allow such class attributes to be retrieved lazily (only imported/set when actually requested via `SomeRecordingInterface.RX` - not dissimilar from the previous `__getattr__` override stuff, but this time for class attributes) but this seems rather nuanced: https://stackoverflow.com/a/13095408

Reply:
OK you can now retrieve all extractor classes as you used to, but they are set only at the time of access

Reply:
See https://github.com/catalystneuro/neuroconv/pull/98 comments for some specific details and descriptions